api/src/main/resources/org/openmrs/api/db/hibernate/Encounter.hbm.xml

```diff
@@ -59,7 +59,7 @@
 			<column name="form_id" />
 		</many-to-one>
 		<!-- bi-directional one-to-many association to Order -->
-		<set name="orders" inverse="true">
+		<set name="orders" inverse="true" where="order_group_id is null">
```

Comment:
Right now the encounter is not having any orderGroup associated to it.  The encounter has orders which internally has orderGroups. And an orderGroup contains orders. 

So this mapping is not needed